File: src/Services/PropertyApi/Models/Property.cs
namespace PropertyApi.Models;
public class Property
{
    public string Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public string LocationDetail { get; set; }
    public string City { get; set; }
    public string Country { get; set; }
    public string Slug { get; set; }
    public float PricePerNight { get; set; }
    public string[] Amenities { get; set; }
    public string HostId { get; set; }
    public Householder Host { get; set; }
    public string TypeId { get; set; }
    public PlaceType Type { get; set; }
    public int NumberOfBethroom { get; set; }
    public int NumberOfBedroom { get; set; }
    public int MaxGuests { get; set; }
    public int AverageRate { get; set; }
    public string CreateAt { get; set; }
}

File: src/Web/SPA/src/app/Shared/Models/property.ts
export type Property = {
  medias: Media[];
  id: string;
  title: string;
  description: string;
  locationDetail: string;
  city: string;
  country: string;
  slug: string;
  pricePerNight: string;
  amenitie: string;
  hostId: string;
  typeId: string;
  numberOfBethroom: number;
  numberOfBedroom: number;
  maxGuests: number;
  averageRate: number;
  createAt: string;
};

export type Media = {
  id: '6d8f838a-3a95-4b72-bceb-58f49bceb302';
  url: 'https://loremflickr.com/640/480/house/any?lock=1015017298';
  isThumb: true;
  propertyId: '9e63728a-4dec-fadd-c5c1-bd6c73cf21e4';
};
